src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms, $transition-color-time;
```

Comment:
```suggestion
    transition: transform $transition-color-time, color $transition-color-time;
```

Reply:
what about color here?)

Reply:
![image](https://user-images.githubusercontent.com/102994838/205305011-e9c761a8-2c06-4009-a044-fa62114128c3.png)
I added it to one variable with a transition duration
OK, I'll redo it.